presto-docs/src/main/sphinx/release/release-0.157.1-t.rst

```diff
@@ -0,0 +1,168 @@
+=================
+Release 0.157.1-t
+=================
+
+Presto 0.157.1-t is equivalent to Presto release 0.157.1, with some additional features and patches.
+
+Security
+--------
+* LDAP Authentication
+
+  
+SQL
+---
+* Support correlated scalar aggregation subqueries
+* Support correlated scalar subqueries
+* Additional prepared statement syntax: DESCRIBE INPUT and DESCRIBE OUTPUT
+
+
+Data Types
+----------
+* Additional Varchar(x) and Char(x) function implementations
+* Additional Decimal functions implementations
+
+Documentation
+-------------
+* Installation of Presto via Presto Admin
+* Presto Tuning Guide
+
+
+Performance
```

Comment:
We should probably mention our work around Query Optimizer (link to that new section in the docs) that includes statistics for Hive connector, automatic join selection, cross join avoidance.